test/concurrent/SynchronizedListTest.ooc

```diff
@@ -10,6 +10,12 @@ use base
 use concurrent
 use unit
 
+applyIndex: static Int = 0
+applyTestFunc := func (value: Int*) {
+	expect(value@, is equal to(applyIndex + 1))
+	applyIndex = applyIndex + 1
+}
```

Comment:
Maybe it's compiled to a static C function, without creating a Closure object for it ?